src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,104 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from 'react-router-dom';
+import axios from 'axios';
+import {ReactComponent as Back} from '../../assets/back.svg';
+import './Subreddit.css'
+
+
+
+function Subreddit() {
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(false);
+    const [info, setInfo] = useState({});
```

Comment:
Als info uiteindelijk een gevuld object wordt, dan zou ik de default waarde aanpassen naar `undefined`. Dan heb je een duidelijker verschil tussen of het al wél of niet is opgehaald. Je check die wat verder naar onder staat wordt dan ook wat simpeler
```suggestion
    const [info, setInfo] = useState();
```